HDSAviz/Interactive_vis_toolkit.ipynb

```diff
@@ -0,0 +1,443 @@
+    "    variables in different text files.\n",
+    "    \n",
+    "    Input :\n",
+    "    S1_Range: Minimum value of sensitivity for first order effects * 0.001, \n",
+    "    ST_Range: Minimum value of sensitivity for total order effects * 0.001\n",
+    "    \n",
+    "    Output:\n",
+    "    Tab based plotting of Graphs. \n",
+    "    \"\"\"\n",
+    "    working_dir = !pwd\n",
+    "    if working_dir != 'C:/HDSAviz_data/' :\n",
+    "        change_directory()\n",
+    "    \n",
+    "    analysis_files = gather_all_analysis_files()\n",
+    "    tabs_dictionary = {}\n",
+    "    \n",
+    "    for i,files in zip(range(len(analysis_files)), analysis_files):\n",
+    "        # cleaning the file names to get a short name to occupy less \n",
+    "        # space in tabs.\n",
+    "        names = files.replace('analysis_', '' ).replace('.txt','')\n",
+    "        \n",
+    "        try :\n",
+    "            tabs_dictionary[i] = Panel(child = makeplot(files, \n",
+    "                                np.array(['S1', 'ST']), \n",
+    "                                np.array([0.001*S1_Range, 0.001*ST_Range])),\n",
+    "                                title = names)    \n",
+    "        except:\n",
+    "            pass\n",
+    "\n",
+    "    \n",
+    "    tabs = Tabs(tabs=tabs_dictionary.values())\n",
+    "    \n",
+    "    try :\n",
+    "        p = show(tabs)\n",
+    "        output_notebook()\n",
+    "    except:\n",
+    "        pass\n",
+    "    \n",
+    "    return p\n",
+    "\n",
+    "def Interact_with_burtin_plots():\n",
+    "    \"\"\"\n",
+    "    This function adds capability to adjust the minimum threshold for \n",
+    "    sensitiity plotting\n",
+    "    \n",
+    "    Output: \n",
+    "    Interactive graph displaying a slider widgets that allow for \n",
+    "    minimum threshold of sensitivity selection\n",
+    "    \"\"\"\n",
+    "    output_notebook()\n",
+    "    slider1 = FloatSlider(value = 1, min = 0.1, max = 10, \n",
+    "                          step=0.1, title = 'S1-Range')\n",
+    "    slider2 = FloatSlider(value = 1, min = 0.1, max = 10, \n",
+    "                          step=0.1, title = 'ST-Range')\n",
+    "    return interact(plot_all_outcomes_burtin, S1_Range=slider1, \n",
+    "                    ST_Range=slider2)\n",
+    "\n",
+    "def short_tabs_demo(S1_Range,ST_Range):\n",
+    "    \"\"\"\n",
+    "    This function plots the burtin type plots for two outcome \n",
+    "    variables in CO and CO2.\n",
+    "    \n",
+    "    Input :\n",
+    "    S1_Range: Minimum value of sensitivity for first \n",
+    "              order effects * 0.001, \n",
+    "    ST_Range: Minimum value of sensitivity for total \n",
+    "              order effects * 0.001\n",
+    "    \n",
+    "    Output:\n",
+    "    Tab based plotting of Graphs (Only two tabs for simplicity \n",
+    "    and sanity check experimentation). \n",
+    "    \n",
+    "    \"\"\"\n",
```

Comment:
I needed to add change_directory() into this function first.  However, once makeplot is updated so that it doesn't require filenames anymore (and instead is passed pandas dataframes directly) then this will not be an issue.